programs/mango-v4/src/instructions/token_deposit.rs

```diff
@@ -92,6 +92,7 @@ impl<'a, 'info> DepositCommon<'a, 'info> {
         // to deposit to close borrows or do other fixes even if the oracle is bad.
         let unsafe_oracle_state = oracle_state_unchecked(
             &AccountInfoRef::borrow(self.oracle.as_ref())?,
+            EMPTY_KEYED_READER_OPT,
```

Comment:
fwiw, I found `EMPTY_KEYED_READER_OPT` more confusing then `None`. It could be `NO_USDC_PYTH_FEED` if you intend this for this particular function only? But it doesn't feel quite right; make a new function? (that can potentially forward to a private function with an optional argument)

Key issue is that oracle_state_unchecked() now has two variants: the one for primary oracles that doesn't allow orca, and the one for fallbacks that does but needs extra info.

Also all instructions that manually read oracles outside of AccountRetriever will not be able to use fallbacks. that includes token_deposit and token_withdraw (at least for some uses)